### ISSUE TITLE
CouchDB connection not using user credentials from URL

### ISSUE THREAD

**Issue by MidnightLightning** [benweet/stackedit#846]
When specifying a CouchDB URL like `https://username:password@example.cloudant.com/stackedit` as the documentation suggests, I can see XHR requests get sent to `https://example.cloudant.com/stackedit`, which is correct, but the username and password aren't used, and so the `POST` request is returning a 401 error.

**Comment by mcescalante** [comment]
Yeah, I'm noticing that when I use `curl` to make a request using `http://username:password@instance.smileupps.com/documents` that my requests work fine, however when I put it into the URL in stackedit, I get a 403 back saying that only administrators can edit design documents.
